k8s/upgrades/README.md

```diff
@@ -308,10 +319,15 @@ spec:
         - "--from-version=1.0.0"
 
         # --to-version is the version desired upgrade version
-        - "--to-version=1.8.0"
+        - "--to-version=1.9.0"
 
-        #VERIFY that you have provided the correct SPC Name
-        - "--spc-name=cstor-sparse-pool"
+        # VERIFY that you have provided the correct SPC Name
+        # The spc-name flag is required till 1.8.0 upgrades
```

Comment:
Lets add comment as:
# Bulk upgrade is supported from 1.9
# To make use of it, please provide the list of SPCs
# as mentioned below
- "cstor-sparse-pool"
- "cstor-disk-pool"
# For upgrades lesser than 1.9.0, use
# '--spc-name=<spc_name> format as
# below commented line
# - "--spc-name=cstor-sparse-pool"